test/core/api/service/author/author_api_test.cpp

```diff
@@ -185,6 +203,223 @@ MATCHER_P(eventsAreEqual, n, "") {
   return (arg.id == n.id) and (arg.type == n.type);
 }
 
+/**
+ @given unsupported KeyTypeId for author_insertKey RPC call
+ @when insertKey called, check on supported key types fails
+ @then corresponing error is returned
+ */
```

Comment:
Please, follow next doc-comment style:
```suggestion
/**
 * @given unsupported KeyTypeId for author_insertKey RPC call
 * @when insertKey called, check on supported key types fails
 * @then corresponing error is returned
 */
```

Reply:
done